### PR TITLE
[15.0][MIG] module_analysis

### PR DESCRIPTION
**Pull request by ghost**
I migrated module_analysis module from 13.0 version to 15.0 version.

### CODE REVIEW

**Review by legalsylvain — COMMENTED** [review]
Hi. Thanks for porting this module ! Could you squach your commit once migration is done to make the diff review easier to do ?
Regards.

File: module_analysis/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Module Analysis",
+    "summary": "Add analysis tools regarding installed modules"
+    " to know which installed modules comes from Odoo Core, OCA, or are"
+    " custom modules",
+    "author": "GRAP, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/server-tools",
+    "version": "15.0.1.1.0",
```
Comment (ivantodorovich) [inline module_analysis/__manifest__.py:12]:
```suggestion
    "version": "15.0.1.0.0",
```

**Review by ivantodorovich — APPROVED** [review]
Minor comment, otherwise seems ok
Code review only

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi. What is the state of this PR ? 
- could you solve conflict ? 
- could you cherry pick that V12 PR ? https://github.com/OCA/server-tools/pull/2058

Thanks !

**Review by legalsylvain — APPROVED** [review]
thanks for the cherry pick !

**Review by cyrilmanuel — APPROVED** [review]
LGTM thank you :)

**Comment by simahawk** [comment]
@juliette-blanc can you please rewrite the commit msg for the migration? 

`[IMP] module_analysis: black, isort, prettier` should be `[MIG] module_analysis: migrate to v15`

**Comment by simahawk** [comment]
> Hi. Thanks for porting this module ! Could you squach your commit once migration is done to make the diff review easier to do ?
> Regards.

@legalsylvain actually this is not desired IMO. The migration commit the the linting commit should stay by themselves to:
1. ease fwd porting of the migration commit
2. ease diff: you'll see only relevant changes from the migration itself

Also, talking about commit messages: would you mind to rewrite yours? Is completely cut...
Should be something like:
```
[FIX+IMP] module_analysis : remove analysis when updating modules

When updating modules, the analysis is partial and makes the update slower.
Instead, use cron task that is executed nightly to update analysis automatically
```

Not a blocker, but nice to have before we merge one of the PRs otherwise is going to become harder to port it (msg won't match if we rewrite in one branch only).

**Comment by rvalyi** [comment]
for the record this v14 migration PR has more commits https://github.com/OCA/server-tools/pull/2085, I'm trying to get it merged...

**Review by sebalix — APPROVED** [review]

**Pull request merged**